sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosMicrometerMetricsOptions.java

```diff
@@ -23,6 +23,8 @@ public final class CosmosMicrometerMetricsOptions extends MetricsOptions {
     private EnumSet<TagName> defaultTagNames = TagName.DEFAULT_TAGS.clone();
     private double[] defaultPercentiles = { 0.95, 0.99 };
     private boolean defaultShouldPublishHistograms = true;
+
+    private boolean defaultApplyDiagnosticThresholdsForTransportLevelMeters = false;
```

Comment:
```suggestion
    private boolean applyDiagnosticThresholdsForTransportLevelMeters = false;
```

Reply:
usually na,ing guideline for fields is to not use a verb + subject - but a adjective or subject, isn't it?

Reply:
Also using the defaultPrefix to match the pattern for other fields. Indicating that this is the default value (which can be overridden in MeterOptions)